src/commands/cmd_zset.cc

```diff
@@ -1285,6 +1345,7 @@ REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandZAdd>("zadd", -4, "write", 1, 1, 1),
                         MakeCmdAttr<CommandZScore>("zscore", 3, "read-only", 1, 1, 1),
                         MakeCmdAttr<CommandZMScore>("zmscore", -3, "read-only", 1, 1, 1),
                         MakeCmdAttr<CommandZScan>("zscan", -3, "read-only", 1, 1, 1),
-                        MakeCmdAttr<CommandZUnionStore>("zunionstore", -4, "write", 1, 1, 1), )
+                        MakeCmdAttr<CommandZUnionStore>("zunionstore", -4, "write", 1, 1, 1),
+                        MakeCmdAttr<CommandZUnion>("zunion", -4, "read-only", 1, 1, 1), )
```

Comment:
Looks like the ZUNIONSTORE command wrongly passed the key position. For the ZUNIONSTORE/ZUNION command, its key positions need to calculate by the [numkeys] argument. Can refer to the `CommandZMPop::Range` implementation.

Reply:
Updated both ZUNION and ZUNIONSTORE